### ISSUE TITLE
Resize background image

### ISSUE THREAD

**Issue by timhaley94** [timhaley94/holdem#30]
The background image doesn't look good on some desktop sizes. We need to dynamically size the image based on the window size so that it looks better. Potentially we need to replace the image altogether.

<img width="1440" alt="Screen Shot 2020-08-06 at 7 38 01 PM" src="https://user-images.githubusercontent.com/14980932/89594084-ed4bae80-d81e-11ea-8890-8d7223e0a216.png">

**Comment by eliaswood** [comment]
I was gonna tackle this but didn't want to resize if it was gonna get blasted. It might also look cool for this with the chat window and the game board if there are some poker chips scattered about the bones or something on them that might fill in space and be easier to make out what it is with the chat and game windows open

**Comment by timhaley94** [comment]
Yeah, that sounds good! I would think we'll end up with different art and the end of the day. Not sure if Laura has time to work on it, so I might need to try my hand at some point. 😅

**Comment by eliaswood** [comment]
Nice I will leave this one for now then

**Comment by timhaley94** [comment]
@prabhatpankaj interested in working on this one? Should be a good, self contained, first issue.
